sdk/graphrbac/graph/src/graphRbacManagementClientContext.ts

```diff
@@ -37,25 +37,25 @@ export class GraphRbacManagementClientContext extends msRestAzure.AzureServiceCl
     if (!options) {
       options = {};
     }
-    if(!options.userAgent) {
+    if (!options.userAgent) {
       const defaultUserAgent = msRestAzure.getDefaultUserAgentValue();
       options.userAgent = `${packageName}/${packageVersion} ${defaultUserAgent}`;
     }
 
     super(credentials, options);
 
-    this.apiVersion = '1.6';
+    this.apiVersion = options.apiVersion || 'v1.0';
```

Comment:
Our clients are autogenerated from a swagger spec. We have a swagger for api version 1.6 over [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json). However, apiVersion `v1.0` is radically different and we don't have a swagger spec for that.

Reply:
@amarzavery Thanks for the information, I'll close this PR then, and open an issue when rest API team releases the new swagger specs for the new API.
Do you have any reference for how to generate the specs for the SDK here?

Reply:
There is a [Microsoft Graph JS sdk](https://github.com/microsoftgraph/msgraph-sdk-javascript) for the v1.0 api version that you want to use.

Reply:
Great, thanks for the help Amar.